tokenizer/characters.go

```diff
@@ -28,7 +28,7 @@ func isIdentifierCharacter(ch rune) bool {
 	return isLetter(ch) || isDigit(ch) || (ch == '_')
 }
 
-func isLeadingIdentifierCharacter(ch rune) bool {
+func IsLeadingIdentifierCharacter(ch rune) bool {
```

Comment:
I'm not thrilled with having to make this function public, but I figure that's better than copy-pasting. Open to feedback though.